### ISSUE TITLE
AVD pattern: deployment wizard bug with the "VM in Separate Resource Group" checked

### ISSUE THREAD

**Issue by jude-chen** [Azure/azure-monitor-baseline-alerts#457]
### Check for previous/existing GitHub issues

- [x] I have checked for previous/existing GitHub issues

### Issue Type?

Bug

### Description

In the deployment wizard, when the "VM in Separate Resource Group" checkbox is checked, it will make the "AVD Resource Group" field invalid and value not populated. 

<img width="446" alt="Image" src="https://github.com/user-attachments/assets/5dfcfa9d-068f-495b-8f6c-49f702741c17" />

what we see when the checkbox is not checked

<img width="464" alt="Image" src="https://github.com/user-attachments/assets/5ac6b9df-ba59-45f8-8d0f-b6735ad0b1be" />

when checked, the "AVD Resource Group" field will be gone

This will result in deployment failure with the below error.

<img width="856" alt="Image" src="https://github.com/user-attachments/assets/6e1602b1-5f9e-49eb-b759-aa502da51dc3" />

And the detailed error message: 
```
{"code":"InvalidResourceGroup","message":"The provided resource group name '<Resource Group Name>' has these invalid characters: '<  >'. The name can only be a letter, digit, '-', '.', '(', ')' or '_'. For more details, visit https://aka.ms/ResourceGroupNamingRestrictions ."}
```

**Comment by JCoreMS** [comment]
There is another issue with the permissions being applied that may or may not be related.  Thanks for bringing this to our attention and hopefully we'll have a fix out very soon.  Stay tuned.

**Comment by JCoreMS** [comment]
Thanks @jude-chen for the input and feedback.  This issue has been addressed with v2.0.0 and the option no longer hides the needed RG for the AVD Resources.
